CONTRIBUTING.md

```diff
@@ -139,7 +74,7 @@ While the prerequisites above must be satisfied prior to having your pull reques
 
 ### Local Setup
```

Comment:
I need to update this file a lot more, but that's outside of the scope of this PR.